lib/graph3d/Graph3d.js

```diff
@@ -322,18 +327,13 @@ Graph3d.prototype.getDistinctValues = function(data, column) {
 
 
 Graph3d.prototype.getColumnRange = function(data,column) {
-  var minMax;
+  var minMax  = new Range();
 
   for (var i = 0; i < data.length; i++) {
     var item = data[i][column];
-
-    if (i === 0) {
-       minMax = { min: item, max: item};
-    } else {
-      if (minMax.min > item) { minMax.min = item; }
-      if (minMax.max < item) { minMax.max = item; }
-    }
+    minMax.add(item);
```

Comment:
It's also a bit weird that a `range`/`minMax` gets an item and sets/adds with it. I would expect to get a range or a coordinate value. Am I missing something?